examples/morpheusvm/tests/generator/mixed.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package generator
+
+import (
+	"context"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/hypersdk/api/jsonrpc"
+	"github.com/ava-labs/hypersdk/auth"
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/codec"
+	"github.com/ava-labs/hypersdk/crypto/bls"
+	"github.com/ava-labs/hypersdk/crypto/ed25519"
+	"github.com/ava-labs/hypersdk/crypto/secp256r1"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/actions"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/vm"
+	"github.com/ava-labs/hypersdk/tests/fixture"
+	"github.com/ava-labs/hypersdk/tests/workload"
+)
+
+var _ workload.TxGenerator = (*multiAuthTxGenerator)(nil)
+
+type multiAuthTxGenerator struct {
+	txCheckInterval     time.Duration
+	addressAndFactories []addressAndFactory
+	balance             uint64
+	count               int
+}
+
+func NewMultiAuthTxGenerator(key *fixture.Ed25519TestKey, txCheckInterval time.Duration) (workload.TxGenerator, error) {
+	blsPriv, err := bls.GeneratePrivateKey()
```

Comment:
i don't know how these bls keys were previously able to send balances. They are randomly generated and I can't seem to find where we allocate their balances.